test/index.js

```diff
@@ -85,7 +85,7 @@ describe('KeyringController', function () {
   });
 
   describe('createNewVaultAndKeychain', function () {
-    it('should set a vault on the configManager', async function () {
+    it('should create a new vault', async function () {
```

Comment:
This change is unrelated; I just happened to notice a reference to a now-extinct "configManager" abstraction in another test. This was the last remaining reference to it.